### PR TITLE
Add BenchmarkSchedulingWaitForFirstConsumerPVs benchmark

### PR DESCRIPTION
**Pull request by mborsz**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
/kind feature
> /kind flake

**What this PR does / why we need it**:
This PR adds a new benchmark that is representative for use case when a lot of pods with PVs attached are created.

Currently kube-scheduler is quite slow for this use case.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by ahg-g** [comment]
The benchmark is testing the path where the scheduler has to do the volume binding, correct?

One nit, otherwise looks good to me for now, but those benchmarks will soon be deprecated in favor of the new benchmark framework: https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler_perf/scheduler_perf_test.go

it would be great if you can add the benchmark under the new suite.

File: test/integration/scheduler_perf/scheduler_bench_test.go [hunk 1]
```diff
@@ -557,6 +576,42 @@ func awsVolumeFactory(id int) *v1.PersistentVolume {
 	}
 }
 
+func gceVolumeFactor(id int) *v1.PersistentVolume {
```
[the remark this answers is not in the document]
Comment (mborsz) [reply test/integration/scheduler_perf/scheduler_bench_test.go]: Thanks

**Comment by mborsz** [comment]
> The benchmark is testing the path where the scheduler has to do the volume binding, correct?
> 
> One nit, otherwise looks good to me for now, but those benchmarks will soon be deprecated in favor of the new benchmark framework: https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler_perf/scheduler_perf_test.go
> 
> it would be great if you can add the benchmark under the new suite.

+1 for using new benchmark framework.
I'm afraid rewriting this to new framework will take some nontrivial time and I don't want to block on this with actual improvements to this path, so how about submitting this change, then submitting actual improvements and then migrating to the new framework?

**Comment by ahg-g** [comment]
/approve 

Sounds good.

**Comment by wojtek-t** [comment]
/approve

@ahg-g - please lgtm if you're happy with it

**Comment by mborsz** [comment]
I created https://github.com/kubernetes/kubernetes/pull/88525 with tiny performance improvement

**Pull request merged**